store/imagestore/remote.go

```diff
@@ -19,9 +19,14 @@ package imagestore
 
 import (
 	"database/sql"
+	"errors"
 	"time"
 )
 
+var (
+	ErrRemoteNotFound = errors.New("remote with the given ACI URL not found")
```

Comment:
I changed my mind :smile: . And to keep a common pattern, when possible, in #3071 I started using the same ErrNotFound approach.